example-web/package.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "example-web",
+  "version": "0.0.1",
+  "scripts": {
+    "web": "expo start --web"
```

```suggestion
    "web": "expo web"
```
Skips metro, much faster!